### ISSUE TITLE
rule: contents-no-position

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#132]
## Rule ID
`contents-no-position`

## What does this rule detect?
`display: contents` generates no box, so positioning properties (`position`, `top`, `right`, `bottom`, `left`, `z-index`) have no effect.

## Example: should warn
```html
<div style="display: contents; position: absolute; top: 10px;">
  <span>child</span>
</div>
```

## Example: should NOT warn
```html
<div style="position: absolute; top: 10px;">
  <span>child</span>
</div>
```

## Related CSS Properties
`display`, `position`, `top`, `right`, `bottom`, `left`, `z-index`, `inset`

## References
- [CSS Display Level 3 §6.7 - display: contents](https://drafts.csswg.org/css-display/#valdef-display-contents): "element itself does not generate any boxes"
- Could be merged with `contents-no-box-props` or kept separate for clarity
